### PR TITLE
Completed css-layout-classes exercise.

### PR DESCRIPTION
**Pull request by ChorongShin**
Screen Shot 2022-01-27 at 1 23 45 PM

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        </div>
+        <div class="column-half">
+          <img src="images/dk.png" alt="Donkey Kong">
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-half">
+          <img src="images/mario.png" alt="Mario">
+        </div>
+        <div class="row-half">
```
Comment (codingcodymiller) [inline css-layout-classes/index.html:72]: What is `row-half`? You only need one row class, which should be `row`

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+              gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981.
+              The franchise consist mainy of two game genres, but also inclues
+              additional spin-off titles of various genres.
+            </p>
+            <p>The game of the first genre are mostly single-screen
+              platform/action puzzle types, featuring Donkey Kong as the opponent
+            in an industrial construction setting. </p>
+          </div>
+        </div>
+        <div class="column-half">
```
Comment (codingcodymiller) [inline css-layout-classes/index.html:64]: I don't see any CSS for the `column-half` class

File: css-layout-classes/index.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+          </div>
+          <div class="row-half">
+            <p>Super Smash Bros. Ultimate received universal acclaim from both critics and players,
+              with some critics calling it the best installment in the series. It received praise for
+              its large amount of content and fine-tuning of existing Smash gameplay elements,
+              although its online mode was widely criticized.
+            </p>
+          </div>
+        </div>
+        <div class="column-one-third">
```
Comment (codingcodymiller) [inline css-layout-classes/index.html:30]: I don't see any CSS for the `column-one-third` class

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ChorongShin** [comment]
Screen Shot 2022-01-27 at 4 40 27 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**